libsrc/c128/mou/c128-1351.s

```diff
@@ -202,20 +163,9 @@ INSTALL:
 
 ;----------------------------------------------------------------------------
 ; UNINSTALL routine. Is called before the driver is removed from memory.
-; No return code required (the driver is removed from memory on return).
+; No return code required (the driver is removed from memory on return).																			 																																	
```

Comment:
Again adding whitespace at end of line.
If your IDE does this, it is clearly broken. That's why I hated Eclipse years ago when I had to use it.

Reply:
Notepadd++... used a compare plugin that apparently went stupid.  Sorry for the issue